src/js/libcs/OpDrawing.js

```diff
@@ -389,8 +391,8 @@ eval_helper.drawconic = function(aConic, modifs) {
     Render2D.preDrawCurve();
 
     var eps = 1e-14; //JRG Hab ih von 1e-16 runtergesetzt
-    var mat = aConic.matrix;
-    var origmat = aConic.matrix;
+    var mat = List.normalizeMax(aConic.matrix);
+    var origmat = JSON.parse(JSON.stringify(mat));
```

Comment:
Why would you want to do that JSON detour? All our CindyScript-typed data structures are treated as immutable in any case, so there should be no reason to make a copy. The [conic transformation](https://github.com/strobelm/CindyJS/blob/927e87cd702d2f48693a1f5acded7ff70d413774/src/js/libcs/OpDrawing.js#L427-L429) creates a new matrix and causes `mat` to refer to this, without affecting the original matrix.